src/edfi-paging-test/edfi_paging_test/reporter/summary.py

```diff
@@ -3,17 +3,38 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-from dataclasses import dataclass, field
-from typing import List
+from dataclasses import dataclass
+from edfi_paging_test.helpers.main_arguments import MainArguments
+from pandas import DataFrame
+import pandas as pd
+import json
 import socket
 
 
 @dataclass
 class Summary:
-    key: str
-    description: str
+    run_name: str
+    run_configration: MainArguments
     machine_name : str = socket.gethostname()
-    resources: List[str] = field(default_factory=lambda: ["all"])
 
     def __post_init__(self):
-        self.resources = (self.resources or ["all"])
+        if(self.run_configration):
+            self.run_configration.resourceList = (self.run_configration.resourceList or ["all"])
+            self.run_configration.contentType = self.run_configration.contentType.value
+            self.run_configration.log_level = self.run_configration.log_level.value
+
+    def get_DataFrame(self) -> DataFrame:
+        """
+        Converts Summary to a DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            A new DataFrame representing Summary
+        """
+
+        df = DataFrame(data=[self])
+        json_summary = json.loads(df.to_json(orient="records"))  # type: ignore
+        df = pd.io.json.json_normalize(json_summary)  # type: ignore
```

Comment:
Got the following warning:

```
C:\source\exchange\Suite-3-Performance-Testing\src\edfi-paging-test\edfi_paging_test\reporter\summary.py:38: FutureWarning: pandas.io.json.json_normalize is deprecated, use pandas.json_normalize instead.
```

I made the suggested change and it is working fine.